queuebacca-core/src/main/java/io/axonif/queuebacca/Subscriber.java

```diff
@@ -141,14 +142,13 @@ public void removeTimingEventListener(TimingEventListener timingEventListener) {
 				Client client,
 				WorkExecutor processingPool,
 				ExceptionResolver exceptionResolver,
-				MessageSerializer messageSerializer,
 				TimingEventSupport timingEventSupport
 		) {
 			ScheduledExecutorService subscriptionScheduler = Executors.newScheduledThreadPool(1, new ThreadFactoryBuilder()
 					.setNameFormat(configuration.getMessageBin().getName() + "-subscription-%d")
 					.build());
 
-			Logger logger = new Logger(configuration.getMessageBin().getName(), messageSerializer);
+			Logger logger = LoggerFactory.getLogger(configuration.getMessageBin().getName());
```

Comment:
Side question about this.  Right now graylog doesn't handle the format of the message bin's well.  It treats something like "pushNotification-processing" as "pushNotification" and "processing".  Can you think of a way we could get around this that isn't too dirty?

Reply:
With my next change processing/recycling will disappear. Can we wait until then? It's actually our code, `AxonifySqsCourier`, that adds the `-` in, but if you change it you'd also be changing all the queue names. It wouldn't make sense to alter the name here specifically for graylog.

There's no way to prevent it from tokenizing on `-`?

Reply:
It can wait.  I'm not sure about changing the tokenization, I'll have to ask Steve but I think thats just something elasticsearch does under the hood unfortunately.